src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -83,11 +83,16 @@ internal class ModelCheckingStrategy(
     }
 
     override fun initializeInvocation() {
-        currentInterleaving.initialize()
         super.initializeInvocation()
+        currentInterleaving.initialize()
```

Comment:
Do you need this change? If "no", please revert. If "yes", please add a comment on why this particular order is important.

Reply:
reverted